API/Cinema.API/Controllers/MovieController.cs

```diff
@@ -34,5 +34,21 @@ public async Task<IActionResult> AddMovie([FromBody] CreateMovieRequest request)
                 return StatusCode(result.ErrorCode, result.Message);
             }
         }
+
+        [HttpPatch(ApiRoutes.Movie.Edit)]
+        public async Task<IActionResult> EditMovie([FromBody] EditMovieRequest request)
+        {
+            var result = await _moviesService.EditMovie(request);
```

Comment:
`MoviesService` does not implement `IMoviesService` - `MoviesService` does not contain method `EditMovie()`